CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.6.0 (2017-06-9)
```

Comment:
OK I understand now. Thanks!
But here as we also have to update the latest date on which we are going to release our Android and on another side we have eligibility thing going on (which is still not confirmed). I will update the version and date and will create a new PR after we get our eligibility pass.